app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -368,31 +422,76 @@ class Identification extends React.Component {
+                    onChange={this.handleRaceChange}
                   />
                   <Form.Check
                     className="pt-2"
-                    type="switch"
+                    type="checkbox"
                     id="american_indian_or_alaska_native"
                     label="AMERICAN INDIAN OR ALASKA NATIVE"
-                    checked={this.state.current.patient.american_indian_or_alaska_native}
-                    onChange={this.handleChange}
+                    checked={!!this.state.current.patient.american_indian_or_alaska_native}
+                    disabled={!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
+                    onChange={this.handleRaceChange}
                   />
-                  <Form.Check className="pt-2" type="switch" id="asian" label="ASIAN" checked={this.state.current.patient.asian} onChange={this.handleChange} />
                   <Form.Check
                     className="pt-2"
-                    type="switch"
+                    type="checkbox"
+                    id="asian"
+                    label="ASIAN"
+                    checked={!!this.state.current.patient.asian}
+                    disabled={!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
+                    onChange={this.handleRaceChange}
+                  />
+                  <Form.Check
+                    className="pt-2"
+                    type="checkbox"
                     id="native_hawaiian_or_other_pacific_islander"
                     label="NATIVE HAWAIIAN OR OTHER PACIFIC ISLANDER"
-                    checked={this.state.current.patient.native_hawaiian_or_other_pacific_islander}
-                    onChange={this.handleChange}
+                    checked={!!this.state.current.patient.native_hawaiian_or_other_pacific_islander}
+                    disabled={!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
+                    onChange={this.handleRaceChange}
+                  />
+                  <Form.Check
+                    className="pt-2"
+                    type="checkbox"
+                    id="race_other"
+                    label="OTHER"
+                    checked={!!this.state.current.patient.race_other}
+                    disabled={!!this.state.current.patient.race_unknown || !!this.state.current.patient.race_refused_to_answer}
+                    onChange={this.handleRaceChange}
+                  />
+                  <Form.Check
+                    className="pt-2"
+                    type="checkbox"
+                    id="race_unknown"
+                    label="UNKNOWN"
+                    checked={!!this.state.current.patient.race_unknown}
+                    disabled={!!this.state.current.patient.race_refused_to_answer}
```

Comment:
Should "Refused to Answer" be available as an option when "Unknown" is selected (and vice versa)? This check makes me thing it would be but it isn't.

Reply:
it currently isn't available as an option after this change: https://github.com/SaraAlert/SaraAlert/pull/689#discussion_r577063171

what is the desired/expected behavior that we want? should I ask PO team?

Reply:
I'm not sure myself so yeah I think we should verify with the PO team what is expected.

Reply:
thread has been started here: https://mitrecorp.slack.com/archives/C01A6P94R9B/p1614144113035600

Reply:
Sounds like it should stay as is. Resolving.